xml/System.Net.Http/HttpClient.xml

```diff
@@ -157,10 +157,11 @@ public class GoodController : ApiController
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If `null` is specified for `handler` parameter, the <xref:System.Net.Http.WebRequestHandler> is used as transport handler.  
+ `handler` will be disposed of by Dispose().  
```

Comment:
@alfredmyers, it would be useful to replace "Dispose()" with a link: `by calling [HttpClient.Dispose](<xref:System.Net.Http.HttpMessageInvoker.Dispose>).`

Reply:
actually @rpetrusha, the correct link should not include the symbols inside the parentheses. I think like this, the build will fail. I think the correct link would look like `[HttpClient.Dispose](xref:System.Net.Http.HttpMessageInvoker.Dispose)`

Reply:
Are there any differences in the way the links should be marked up between lines 160 and 190?
One seems to be markdown embeded in XML. The other plain XML.
Is that XML based on [XML Documentation Comments](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/xml-documentation-comments)?

Reply:
@alfredmyers, actually, @mairaw was correct, and I confused the syntax of an API link whose title text is different than the link. Both should be `[HttpClient.Dispose](xref:System.Net.Http.HttpMessageInvoker.Dispose)`. Thanks, by the way, for making all of the additional changes.